synapse/tests/test_cortex.py

```diff
@@ -813,6 +814,17 @@ def onAdd(node):
 
     async def test_splice_generation(self):
 
+        def isin(mesg, splices):
```

Comment:
we're already poping user and time out of the splice; why not just pop pprov as well?

Reply:
you read my mind. already done.